prometheus_juju_exporter/config_default.yaml

```diff
@@ -13,15 +13,14 @@ exporter:
   collect_interval: 15
 
 detection: # parameters affecting the detection algorithm
-  skip_interfaces: [] 
-  # This option can be used to exclude interfaces that might confuse the detection algorithm.
-  # Takes a list of regexps as input. Any interface name matching any of the provided expressions will be skipped.
-  # For example: skip_interfaces: ["br.*", "tap-.*"] 
-  virt_macs:
-    - "52:54:00"
-    - "fa:16:3e"
-    - "06:f1:3a"
-    - "00:0d:3a" # Microsoft
-    - "00:50:56" # VMware
+  match_interfaces: .*
```

Comment:
I would have a slight preference for defaulting to `''`. `.*` is a valid regex so it's kind of a given that you could specify it here if desired.

Reply:
Sounds good. I've updated it.

Reply:
Would you want the empty default `''` to have a different behavior than `.*`? Because I actually think that this is better because it's more explicit about the default behavior.
It ultimately behaves the same way as previously `[]` did with `skip_interfaces`.

Reply:
no, I think `''` should be equivalent to `.*`. It's documented in the config so we should be ok